src/components/10-atoms/button/README.md

```diff
@@ -93,3 +102,7 @@ Based on the string-valued attribute `icon`, interpreted as icon name, an icon w
 ### onClick
 
 The function-valued attribute `onClick` can be used as a callback prop for React and other frameworks.
+
+### Migration version 1 to 2
```

Comment:
very very nice! we should have a migration guide section fixed in our scaffolding:

Reply:
#1155

Reply:
Correct Button will get new major :)